### PR TITLE
Fix Spelling Errors

### PR DESCRIPTION
**Pull request by greggman**

### CODE REVIEW

**Comment by greggman** [comment]
Note: I use VSCode and in VSCode I use the [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension. This makes spelling errors stick out. I also ends up making me name things so they don't show up as errors. For example 

```
const timems = performance.now();   // timems is not word
const timeMS = performance.now();   // ok? 
const timeMs = performance.now();   // ok?
```

I get those rules are kind of arbitrary although many style guides would probably ban `timems` under their camelCase or snake_case rules.

Anyway, my point is only that I've found it helps my code.

File: src/webgpu/web_platform/copyToTexture/canvas.spec.ts [hunk 1]
```diff
@@ -662,8 +662,8 @@ g.test('copy_contents_from_gpu_context_canvas')
   Then call copyExternalImageToTexture() to do a full copy to the 0 mipLevel
   of dst texture, and read the contents out to compare with the canvas contents.
 
-  Provide premultiplied input if 'premultipliedAlpha' in 'GPUImageCopyTextureTagged'
-  is set to 'true' and unpremultiplied input if it is set to 'false'.
+  Do premultiply alpha during copy if  'premultipliedAlpha' in 'GPUImageCopyTextureTagged'
+  is set to 'true' and do unpremultiply alpha if it is set to 'false'.
```
Comment (kainino0x) [inline src/webgpu/web_platform/copyToTexture/canvas.spec.ts:666]: Merge conflict here?
```suggestion
  Provide premultiplied input if 'premultipliedAlpha' in 'GPUImageCopyTextureTagged'
  is set to 'true' and unpremultiplied input if it is set to 'false'.
```

**Review by kainino0x — APPROVED** [review]
This is excellent! thank you!

File: src/webgpu/web_platform/copyToTexture/canvas.spec.ts [hunk 2]
```diff
@@ -662,6 +662,7 @@ g.test('copy_contents_from_gpu_context_canvas')
   Then call copyExternalImageToTexture() to do a full copy to the 0 mipLevel
   of dst texture, and read the contents out to compare with the canvas contents.
 
+  Do premultiply alpha during copy if  'premultipliedAlpha' in 'GPUImageCopyTextureTagged'
```
Comment (kainino0x) [inline src/webgpu/web_platform/copyToTexture/canvas.spec.ts:665]:
```suggestion
```
oops

**Pull request merged**